symposion/schedule/admin.py

```diff
@@ -61,13 +65,25 @@ class PresentationAdmin(admin.ModelAdmin):
         'description',
         'abstract',
         'section__name',
+        'video_url',
+        'assets_url',
+        'slides_url',
     )
 
+    def get_queryset(self, request):
+        qs = super(PresentationAdmin, self).get_queryset(request)
+        return qs.select_related('speaker', 'speaker__user',
+                                 )
```

Comment:
I think that the trailing parenthesis would be better read on the previous line.